a.rb

```diff
@@ -3,4 +3,14 @@ class Error < Exception
 
 raise Error
 
-puts "Hello world"
+def write(message, io=STDOUT)
+  io.puts(message)
+end
+
+puts "hogehoge"
+
+io = File.open("foo.txt")
+
+write "Hello world", io
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.